app/javascript/components/shared/Pagination.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Pagination as PaginationSematicUi } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import PaginationButton from './PaginationPrevButton';
+
+export default function Pagination({
+  page, totalPages, setPage, borders,
+}) {
+  const handlePage = (e, { activePage }) => {
+    const gotopage = { activePage };
+    const pagenum = gotopage.activePage;
+    setPage(pagenum);
+  };
```

Comment:
```suggestion
  const handlePage = (e, { activePage }) => { setPage(activePage); };
```

Reply:
This is from a previous PR, will this work @hadicheaito1

Reply:
i think you would need to do `const handlePage = (e, { activePage }) => { setPage(activePage.activePage); };`. It might not work though since right now `gotopage = {activePage} `and not just `activePage`

Reply:
Amir was right.